packages/theme-compatibility/src/base/editor.scss

```diff
@@ -53,6 +53,17 @@
 	}
 
 	.crowdsignal-forms-question-wrapper__content {
+
+		&.is-horizontal {
+			display: block; // abort flex to get full width!
```

Comment:
Or `flex: 1;` ? Also, why is this necessary here? Shouldn't the styled component styles take care of it already?